editoast/src/views/train_schedule.rs

```diff
@@ -563,7 +563,7 @@ fn build_simulation_request(
                 .stop_for
                 .as_ref()
                 .map(|t| t.num_milliseconds() as u64),
-            on_stop_signal: schedule_item.on_stop_signal,
+            reception_signal: schedule_item.reception_signal.clone(),
```

Comment:
Wdyt about making the enum derive `Copy`?

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/8983/commits/daf83e3461ef97280de9964d6ba7202e7260bfd3